src/templates/api.js

```diff
@@ -98,7 +98,7 @@ class Template extends Component {
 
           return {
             file,
-            title: pageNode.meta.title,
+            title: pageNode.meta.title.replace('API', '').trim(),
```

Comment:
quick fix: remove the words API from the sidebar so it doesn't say API next to literally every line